### ISSUE TITLE
MultiWayIf

### ISSUE THREAD

**Issue by georgefst** [fourmolu/fourmolu#52]
```hs
bar x y =
    if
            | x > y -> x
            | x < y ->
                y
            | otherwise -> x
```

This is obviously way too much indentation.

Somehow we've deviated from Ormolu's [output](https://github.com/tweag/ormolu/blob/master/data/examples/declaration/value/function/multi-way-if.hs). And seemingly during the period when we didn't have proper tests in place. Will need to do some `git bisect`-ing.

**Comment by expipiplus1** [comment]
```haskell
diff --git a/src/Ormolu/Printer/Meat/Declaration/Value.hs b/src/Ormolu/Printer/Meat/Declaration/Value.hs
index ced5c3b..6a36288 100644
--- a/src/Ormolu/Printer/Meat/Declaration/Value.hs
+++ b/src/Ormolu/Printer/Meat/Declaration/Value.hs
@@ -677,8 +677,8 @@ p_hsExpr' s = \case
     p_if exprPlacement p_hsExpr if' then' else'
   HsMultiIf NoExtField guards -> do
     txt "if"
-    breakpoint
-    inci . inci $ sep newline (located' (p_grhs RightArrow)) guards
+    space
+    sitcc $ sep newline (located' (p_grhs RightArrow)) guards
   HsLet NoExtField localBinds e ->
     p_let p_hsExpr localBinds e
   HsDo NoExtField ctx es -> do
```

Seems to do what I'd expect for two space indentation, for four space indentation it perverts the AST :(

**Comment by brandonchinn178** [comment]
Personally, I'd much rather prefer the common case to be one of:
```hs
-- two space
if
  | x > y -> x
  | x < y -> y
  | otherwise -> x

-- three space
if
   | x > y -> x
   | x < y -> y
   | otherwise -> x

-- four space
if
    | x > y -> x
    | x < y -> y
    | otherwise -> x
```
or
```hs
-- two space
if  | x > y -> x
    | x < y -> y
    | otherwise -> x

-- three space
if | x > y -> x
   | x < y -> y
   | otherwise -> x
```

**Comment by brandonchinn178** [comment]
It seems like the common case is fixed in Ormolu 0.7 / Fourmolu 0.13? https://github.com/fourmolu/fourmolu/pull/324/commits/2ee0dab98d9b1e0aae13b7845f659e184024b494

https://github.com/fourmolu/fourmolu/blob/main/data/examples/declaration/value/function/multi-way-if-four-out.hs
